libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.ts

```diff
@@ -412,16 +419,56 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
     this.#changeDetector.markForCheck();
   }
 
+  public onCountryFieldFocusout({ relatedTarget }: FocusEvent): void {
+    if (this.inputBoxHostSvc && relatedTarget) {
+      if (!this.inputBoxHostSvc.focusIsInInput(relatedTarget)) {
+        this.toggleCountrySearch(false);
+      }
+    } else {
+      if (!this.#elementRef.nativeElement.contains(relatedTarget)) {
+        this.toggleCountrySearch(false);
+      }
+    }
+  }
+
   public countrySearchAnimationEnd(e: AnimationEvent): void {
     if (!this.countrySearchShown) {
       this.phoneInputShown = true;
     } else {
       this.#adapterService.focusCountrySearchElement(e.element);
+
+      // add event listeners for focusout from the buttons on either side of country search field.
+      let countryFlagButton = this.#elementRef.nativeElement.querySelector(
+        'button.sky-phone-field-country-select-btn',
+      );
+      let dismissCountrySearchButton =
+        this.#elementRef.nativeElement.querySelector(
+          'button.sky-phone-field-search-btn-dismiss',
+        );
+      if (this.inputBoxHostSvc) {
+        countryFlagButton = this.inputBoxHostSvc.queryHost(
+          'button.sky-phone-field-country-select-btn',
```

Comment:
the internal function `queryHost` was added to input box and input box host svc for this query